nameko/amqp.py

```diff
@@ -38,12 +39,20 @@ class TestTransport(Transport):
     Connection = ConnectionTester
 
 
-def verify_amqp_uri(amqp_uri):
-    connection = Connection(amqp_uri)
+def verify_amqp_uri(amqp_uri, config=None):
```

Comment:
It's odd to pass the `amqp_uri` and the whole `config` object (which contains the AMQP URI data).

I would prefer if the arguments here were `amqp_uri` and `ssl_params` (or just `ssl`)